src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -69,7 +69,7 @@ public ResponseEntity<Object> createNewSubmission(
 
         try {
             if (isValidationEnabled) {
-                var validationErrors = overseasEntitySubmissionDtoValidator.validate(overseasEntitySubmissionDto, new Errors(), requestId);
+                var validationErrors = overseasEntitySubmissionDtoValidator.validateFull(overseasEntitySubmissionDto, new Errors(), requestId);
```

Comment:
I think we should be calling 'validatePartial' on or around line 168 of the controller - so the other POST 'start' method.

Reply:
yes , the /start endpoint should call validatePartial and the original / Post endpoint should call validateFull - looks like the partial call has been added to the /start now

Reply:
Ok added that call and the uts for this.